ocs_ci/framework/main.py

```diff
@@ -58,9 +58,9 @@ def init_ocsci_conf(arguments=None):
         ) as file_stream:
             custom_config_data = yaml.safe_load(file_stream)
             framework.config.update(custom_config_data)
-    ocs_registry_image = args.ocs_registry_image if (
-        args.ocs_registry_image
-    ) else framework.config.DEPLOYMENT.get('ocs_registry_image')
+    ocs_registry_image = args.ocs_registry_image or (
+        framework.config.DEPLOYMENT.get('ocs_registry_image')
+    )
```

Comment:
I personally prefer something less python-idiomatic but more explicit and possibly less error-prone like this:
```suggestion
    ocs_registry_image = framework.config.DEPLOYMENT.get('ocs_registry_image')
    if args.ocs_registry_image:
        ocs_registry_image = args.ocs_registry_image
```
But that's more a taste thing. ;-)